### PR TITLE
Avoid Intent redirect on Runtime Banner for the DEV Android app context

### PR DESCRIPTION
**Pull request by fdocr**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

The DEV Android app was decommissioned but we're unable to ship an updated build right now. So we're handling the Runtime Banner as the backup to try and migrate users to the Forem mobile app.

## QA Instructions, Screenshots, Recordings

I'd recommend not trying to QA locally (somewhat tricky), I tested this by doing:
1. Boot DEV Android app locally and point to an ngrok endpoint (local forem running this branch)
2. Check the banner opens the Google Play Store when tapped

### UI accessibility concerns?

None - All UI elements used weren't modified

## Added/updated tests?

- [x] No, and this is why: Edge case where we're handling the now decommissioned DEV Android app

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: [A public announcement](https://dev.to/devteam/sunsetting-the-devto-ios-app-44nl) was already made a while ago. This is only improving the current experience

## [optional] Are there any post deployment tasks we need to perform?

None

## [optional] What gif best describes this PR or how it makes you feel?

installation wizard

### CODE REVIEW

File: app/javascript/runtimeBanner/RuntimeBanner.jsx [hunk 1]
```diff
@@ -22,6 +22,13 @@ function removeFromDOM() {
...
+  if (navigator.userAgent === 'DEV-Native-android') {
+    // The DEV Android app has been decommissioned and it kept a custom UA.
+    // Instead of a custom intent we are redirecting directly to the Play Store
+    // because the app isn't capable of handling these
+    return FOREM_GOOGLE_PLAY_URL;
+  }
+
   return (
     'intent://scan/#Intent;' +
     'action=android.intent.action.SEND;' +
```
Comment (fdocr) [inline app/javascript/runtimeBanner/RuntimeBanner.jsx:34]: The DEV Android app isn't capable of handling Intents so we should rely on redirecting the app to the Google Play Store instead. If the app is already installed they'll be presented with the option to open the app from there.

**Review by rt4914 — APPROVED** [review]
From code it looks good. Haven't tested it locally.

**Review by aitchiss — APPROVED** [review]
Code change makes sense to me 👍
